concepts/symbols/about.md

```diff
@@ -0,0 +1,80 @@
+# About
+
+[Symbols][symbols] are named identifiers that can be used to refer to a value.
+Symbols are created through a symbol literal, which is by prefixing a name with a `:` character, e.g. `:foo`.
+They also allow for being written as strings, e.g. `:"foo"`, which allows for example spaces in the name.
+
+```ruby
+:foo # => :foo
+:"foo boo" # => :"foo boo"
+```
+
+Symbols are used in many places in the language, including as keys in hashes, to represent method names and variable names.
+
+## Identifier
+
+What makes symbols different from strings is that they are identifiers, and not data or text.
+This means that two symbols with the same name are always the same object.
+
+```ruby
+"foo".object_id # => 60
+"foo".object_id # => 80
+:foo.object_id # => 1086748
+:foo.object_id # => 1086748
+```
+
+## Modifying Symbols
+
+Symbols are immutable, which means that they cannot be modified.
+This means that when you "modify" a symbol, you are actually creating a new symbol.
+There are a few methods that can be used to modify symbols, but they all return new symbols.
+All methods can be found in the [Symbol API][symbols-api].
+
+```ruby
+:foo.upcase # => :FOO
+
+:foo.object_id # => 1086748
+:foo.upcase.object_id # => 60
+```
+
+The benefit of symbols being immutable is that they are more memory efficient than strings, but also safer to use as identifiers.
+
+## Conversion
+
+Symbols can be converted to strings and vice versa.
+This can be useful when you want to modify a symbol, or when you want to use a symbol as a string.
```

Comment:
Since we can not modify a symbol, this should be stated differently.

Reply:
I am inclining here that if you want to modify a symbol, you can convert it to a string, and when you have a string you can modify it. Does that need to be clarified?

Reply:
Yeah, you really can not modify a symbol.  So converting it to a string, modifying that string, and then coercing the string to a symbol to create a new symbol is what is happening.  It is a different object, as can be seen by using `.object_id` to check/verify.  No modification of a symbol is possible.